src/utils/inline-styles.js

```diff
@@ -57,15 +59,18 @@ export default class InlineStyles extends LitElement {
       return;
     }
     const root = this._rootNode || this.ownerDocument;
-    const referenceCount = (referenceCounts.get(root) | 0) - 1;
+    const info = referenceCounts.get(root) || {};
+    const referenceCount = (info.referenceCount || 0) - 1;
```

Comment:
These two lines 62 + 63 could be transformed to a class method because of reuse at line 27. (and if you do that, dont forget to unit test that :) )

Reply:
I decide not too, since it seems too small and the offsets are different (+1 vs -1).